src/pages/VenuePage/VenuePage.tsx

```diff
@@ -65,30 +67,32 @@ const VenuePage = () => {
     return <>This venue does not exist</>;
   }
 
-  if (eventRequestStatus && !event) {
-    return <>This event does not exist</>;
-  }
+  if (!isUserVenueOwner) {
+    if (eventRequestStatus && !event) {
+      return <>This event does not exist</>;
+    }
 
-  if (!event || !venue || !users || !userPurchaseHistoryRequestStatus) {
-    return <>Loading...</>;
-  }
+    if (!event || !venue || !users || !userPurchaseHistoryRequestStatus) {
+      return <>Loading...</>;
+    }
 
-  if (
-    (event.price > 0 &&
-      userPurchaseHistoryRequestStatus &&
-      !hasUserBoughtTicket) ||
-    isEventFinished
-  ) {
-    return <>Forbidden</>;
-  }
+    if (
+      (event.price > 0 &&
+        userPurchaseHistoryRequestStatus &&
+        !hasUserBoughtTicket) ||
+      isEventFinished
+    ) {
+      return <>Forbidden</>;
+    }
 
-  if (!canUserJoinTheEvent(event)) {
-    return (
-      <CountDown
-        startUtcSeconds={event.start_utc_seconds}
-        textBeforeCountdown="Bar opens in"
-      />
-    );
+    if (!canUserJoinTheEvent(event)) {
+      return (
+        <CountDown
+          startUtcSeconds={event.start_utc_seconds}
+          textBeforeCountdown="Bar opens in"
+        />
+      );
+    }
   }
 
```

Comment:
would be nice to include a div at the top of the page saying something like "This is a preview of your venue during an event" in big bold letters. What do you think?

Reply:
good idea